velox/docs/functions/spark/math.rst

```diff
@@ -214,6 +214,13 @@ Mathematical Functions
 
     Returns the modulus (remainder) of ``n`` divided by ``m``. Corresponds to Spark's operator ``%``.
 
+.. spark:function:: rint(x) -> double
+
+    Returns the double value that is closest in value to the argument and is 
```

Comment:
Would you document behavior for NaN and infinity inputs? Let's also add tests for these.

Reply:
Updated, thanks.